manager/cve_handler.py

```diff
@@ -381,7 +382,9 @@ def handle_get(cls, **kwargs):
                     advisories_list = query.dicts()[0]['advisories_list']
                     system_advisories, patch_access = cls._get_advisories_per_system(advisories_list, inv_ids)
                     if 'advisory' in args and args['advisory']:
-                        system_advisories_filtered = {sys: advisories for sys, advisories in system_advisories.items() if args['advisory'] in advisories}
+                        system_advisories_filtered = {sys: advisories for sys, advisories in system_advisories.items() for advisory in advisories if
+                                                      advisory.find(args['advisory']) != -1}
```

Comment:
Why not to use `if args['advisory'] in advisory`? It's more optimized in Python than `find`.

Reply:
Because of the behaviour of `if args['advisory'] in advisory` it's an exact match, but we need to match but parts, for example, '2020' - will match all advisories included 2020 in their name

Reply:
I think this is not correct. Yes, in original implementation it filters only exact matches but you've added another `for` cycle for looping through advisories. `args['advisory'] in advisory` can be used there for finding substring instead of `find`.

Reply:
Also few lines above is call of `query = apply_filters(query, args, [filter_types.ADVISORY])`. Do I understand correctly it filters in DB? Why it needs to be filtered in DB and code?

Reply:
Yep, I see, I will change it 
Regarding DB filtering, it was done to reduce the amount of request, in case there will be CVE which is not including our advisory in `advisory_list`, so the whole CVE will be rejected, so we don't need to send a request for each advisory in advisory_list for this CVE